### PR TITLE
Docs: Session Timeout Config

### PR DESCRIPTION
**Pull request by brandonSc**
Documented new session timeout value in buildkit. I've also broken down the buildkit config sections into smaller subsections so it's easy to document and link to individual configs within those entires.

### CODE REVIEW

File: docs/earthly-config/earthly-config.md [hunk 1]
```diff
@@ -137,16 +137,40 @@ Set this configuration to a lower value if your machine is resource constrained
...
+The `--userns` flag can be set as follows:
 
 ```yaml
 global:
   buildkit_additional_args: ["--userns", "host"]
 ```
 
+#### Session Timeout
+
+By default, Buildkit will automatically cancel sessions (i.e. a single build) after 24 hours.
```
Comment (alexcb) [inline docs/earthly-config/earthly-config.md:154]:
```suggestion
By default, Buildkit will automatically cancel sessions (i.e. individual builds) after 24 hours.
```

**Review by alexcb — APPROVED** [review]

File: docs/earthly-config/earthly-config.md (hunk 1 above)
Comment (alexcb) [reply docs/earthly-config/earthly-config.md:154]: alternatively, switch to `(e.g. a single build)`.

**Comment by brandonSc** [comment]
Oops, I always forget to open my PR at `docs-0.7`

File: docs/earthly-config/earthly-config.md (hunk 1 above)
Comment (alexcb) [reply docs/earthly-config/earthly-config.md:154]: feels weird having `sessions` plural, but id est singular.

**Pull request closed without merging**